.travis.yml

```diff
@@ -6,10 +6,6 @@ script: bash -ex .travis-opam.sh
 sudo: true
 env:
   matrix:
-  - PACKAGE="decompress" OCAML_VERSION=4.03 TESTS=true
```

Comment:
It seems to me that keeping a test for the earliest version we claim to support in the `opam` file would make sense?